helix-core/src/main/java/org/apache/helix/monitoring/mbeans/ClusterStatusMonitor.java

```diff
@@ -937,8 +937,7 @@ private void unregisterAllJobs() {
     }
   }
 
-  // For test only
-  protected ResourceMonitor getResourceMonitor(String resourceName) {
+  public ResourceMonitor getResourceMonitor(String resourceName) {
```

Comment:
any particular reason to make it public?

Reply:
The function need to be accessed by tests in another package. So changed it to public.